README.md

```diff
@@ -1,2 +1,43 @@
-# Website
-This repo is work in progress for the official website of **[Kubesimplify](https://kubesimplify.github.io/website/)**
+<h1 align="center"><a href="https://kubesimplify.github.io/website/">Kubesimplify Website </a></h1>
+<p align="center"><i>The source code for Kubesimplify.com</i></p>
+
+This repository contains the assets required to build the Kubernetes website and documentation. We're glad that you want to contribute! 
+
+- [Code of Conduct ](code-of-conduct.md)
+
+<h1>Contributing Guidelines: </h1>
+
+Kubesimplfy's new website. This codebase is what runs on [Kubesimplify](https://kubesimplify.github.io/website/). 
+
+
+For new developers getting started, run the following in your terminal:
+
+Fork the repo by clicking the fork button. This will clone the repo in your account, changes done here will be shown in the main repo. 
```

Comment:
```suggestion
Fork the repo by clicking the fork button. This will create a copy of the repo in your account, changes done here will not be shown in the main repo. 
```

@anubha-v-ardhan is the wording correct, if not please optimize

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
@anubha-v-ardhan can you check this too